doc/api/child_process.md

```diff
@@ -146,6 +146,9 @@ exec('"my script.cmd" a b', (err, stdout, stderr) => {
 <!-- YAML
 added: v0.1.90
 changes:
+  - version: REPLACEME
+    pr-url: ???
+    description: allow `options.cwd` receive a `URL`.
```

Comment:
```suggestion
    description: The `cwd` option can be a WHATWG `URL` object using
                 `file:` protocol.
```

Reply:
Should it really say "can be a WHATWTG `URL` object"? A `URL` object (capital letters) is always a WHATWG URL. In addition the `fs` APIs allow URL's but almost never mention WHATWG, see https://nodejs.org/api/fs.html

To me it feels to detailed.

Reply:
I've copied this precisely from `fs.md`:

https://github.com/nodejs/node/blob/d28eedba3a789e6ab2b54fc41316f9a0c12d7038/doc/api/fs.md#L1352-L1353

Reply:
Ah, sorry, I think this was on the actual docs but see now that it is meta-data.